app/views/reading_groups/_form.html.erb

```diff
@@ -17,6 +16,13 @@
         <%= f.text_area :description, class: 'text-area form-control' %>
       </div>
     </div>
+    <div class="form-row">
+      <div class="field-container checkbox">
+        <label>
+          <%= check_box :attributes, :public %> <%= t('should_be_public') %>
```

Comment:
I think we should not allow the user to hide his ReadingGroups. This is a feature just for administrators to remove "test" configurations that pollute the system.

The objective of Mezuro is sharing software metrics usage.

What do you think?